index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Practice On Responsive Web Design</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700;900&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <header>
+      <img src="./images/My Logo.png" alt="" class="logo" />
+      <nav class="links">
+        <ul class="main">
+          <li class="first"><a href="#">Portfolio</a></li>
+          <li class="second"><a href="#">About</a></li>
+          <li class="third"><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <section class="container">
+      <h2 class="headin">I’m Brenda.Glad to see you!</h2>
+      <p class="paragrap">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hesitate to contact me.
+      </p>
+      <h3>LET’S CONNECT</h3>
+      <div class="second-container">
+        <img src="./images/Vector (1).png" alt="" class="vector1">
+        <img src="./images/Vector (2).png " alt="" class="vector2">
+        <img src="./images/Vector (3).png" alt="" class="vector3">
+        <img src="./images/Vector (4).png" alt="" class="vector4">
+        <img src="./images/Vector (5).png" alt="" class="vector5">
+        <img src="./images/Vector (6).png" alt="" class="vector6">
+      </div>
+    </section>
```

Comment:
- [ ] The has all the HTML elements but the style is missing the background image. Kindly add the template background image as an `background-image` property, since this is not part of the page's content. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.

Reply:
I have done the needful